src/commands/tx/cosmos.rs

```diff
@@ -14,7 +14,7 @@ use std::process::exit;
 #[derive(Command, Debug, Clap)]
 pub enum Cosmos {
     SendToEth(SendToEth),
-    
+
```

Comment:
Same here, let's delete blank lines added by removing trailing whitespace in enums, structs, etc.

Reply:
We may continue to run into this using `cargo fmt` and `cargo fix`.